src/Chapter1.hs

```diff
@@ -619,8 +631,8 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = let last2 = (abs n) `mod` 100 in (last2 `div` 10) + (last2 `mod` 10) 
```

Comment:
Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1: 

Also, you can remove `()`, they are redundant in this case :slightly_smiling_face: 
```suggestion
sumLast2 n = let last2 = abs n `mod` 100 in (last2 `div` 10) + (last2 `mod` 10) 
```
One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
let last2 = abs n `mod` 100 
    (sndLastDigit, lastDigit) = last2 `divMod` 10
in sndLastDig + lastDig
```

You can see how we could pattern match on the pair 🙂